src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -38,8 +38,9 @@ public class AddressBookParserTest {
     @Test
     public void parseCommand_add() throws Exception {
```

Comment:
Should this method name be changed to `parseCommand_create()`?

Reply:
Yes it should be! Thanks for spotting it!